src/utils/datetime_utils.js

```diff
@@ -0,0 +1,10 @@
+export function getDateTime(timestamp) {
```

Comment:
Also, there seems to be a `timezone_names.js` file that has functionality similar to this file, so you should move the function in that file into this file and make this file the centralized util file that handles anything 'time' related.